pages/conference/_eventType/_id.vue

```diff
@@ -0,0 +1,329 @@
+<template>
+    <i18n-page-wrapper class="px-8 sm:px-10 md:px-32 lg:px-60" custom-x>
+        <core-h1 :title="data.title" center class="font-black"></core-h1>
+        <div class="speech__speakers">
+            <div
+                v-for="(speaker, index) in data.speakers"
+                :key="`speech_details_speaker_${index}`"
+                class="speech_speakerContainer"
+            >
+                <div class="speech__speakerThumbnail">
+                    <img :src="speaker.thumbnail_url" :alt="speaker.name" />
+                </div>
+                <p class="speech__speakerName">{{ speaker.name }}</p>
+            </div>
+        </div>
+
+        <div class="speech__infoSection">
+            <div class="speech__infos">
+                <div class="speech__info">
+                    <img :src="icons.location" class="mx-2" />
+                    {{ locationMapping[data.location] }}
+                </div>
+                <div class="speech__info">
+                    <img :src="icons.language" class="mx-2" />
+                    {{ $t(`languages.${data.language}`) }}
+                </div>
+                <div class="speech__info">
+                    <img :src="icons.datetime" class="mx-2" />
+                    {{ $t(data.dateTag) }} • {{ getTime(data.begin_time) }}-{{
+                        getTime(data.end_time)
+                    }}
+                </div>
+                <div class="speech__info">
+                    <img :src="icons.level" class="mx-2" />
+                    {{ $t(`levels.${data.python_level}`) }}
+                </div>
+                <div class="speech__info">
+                    <img :src="icons.category" class="mx-2" />
+                    {{ $t(`categories.${data.category}`) }}
+                </div>
+            </div>
+        </div>
+
+        <tabs class="speech__tabs">
+            <tab :title="$t('terms.speech')">
```

Comment:
Although this PR doesn't include any `Tab` component's changes, the font-weight needs to be bolder too.

| Actual | Expected |
|:-------|:----------|
| <img width="400" alt="Screen Shot 2021-08-06 at 6 30 15 AM" src="https://user-images.githubusercontent.com/7352613/128429572-2cd2d997-20d7-4569-b86f-575c8be542c1.png"> | <img width="400" alt="Screen Shot 2021-08-06 at 6 30 47 AM" src="https://user-images.githubusercontent.com/7352613/128429596-91d0bd4d-5df3-4daa-bd07-36f691252b6a.png"> |

Reply:
This will be fixed in #131. Resolve for now.